code/__DEFINES/dcs/signals/signals_object.dm

```diff
@@ -336,6 +336,15 @@
 ///sent to targets during the process_hit proc of projectiles
 #define COMSIG_PELLET_CLOUD_INIT "pellet_cloud_init"
 
+// /obj/vehicle/sealed/car/vim signals
+
+///from /datum/action/vehicle/sealed/noise/chime
+#define COMSIG_VIM_CHIME_USED "vim_chime_used"
+///from /datum/action/vehicle/sealed/noise/buzz
+#define COMSIG_VIM_BUZZ_USED "vim_buzz_used"
+///from /datum/action/vehicle/sealed/headlights/vim
```

Comment:
It's not consistent, but there is a pattern to signal definitions.
Goes from the procpath that triggers it, or a description of what event triggers it, then a :, and ()
You insert the extra args that get passed along with the signal into the (), or leave it blank if there are none.

Reply:
changed in the latest commit